server/form-pages/utils/questions.ts

```diff
@@ -11,74 +11,28 @@ export const getQuestions = (name: string) => {
         },
       },
     },
-    'funding-information': {
-      'funding-source': {
-        fundingSource: {
-          question: `How will ${name} pay for their accommodation and service charge?`,
+    'address-history': {
+      'previous-address': {
+        hasPreviousAddress: {
+          question: `Did ${name} have an address before entering custody?`,
           answers: {
-            personalSavings: 'Personal money or savings',
-            benefits: 'Benefits',
-            both: 'Both',
+            yes: 'Yes',
+            no: 'No fixed address',
           },
         },
-      },
-      'national-insurance': {
-        nationalInsuranceNumber: {
-          question: `What is ${name}'s National Insurance number? (Optional)`,
-          hint: 'We need this to set up a Universal Credit and Housing Benefit claim if required.',
-        },
-      },
-      identification: {
-        idDocuments: {
-          question: `What identification documentation (ID) does ${name} have?`,
-          hint: 'Select all that apply.',
+        knownAddress: {
+          question: 'What was the address?',
           answers: {
-            passport: 'Passport',
-            travelPass: 'Travel pass with photograph',
-            birthCertificate: 'Birth certificate',
-            bankOrDebitCard: 'Bank account or debit card',
-            bankStatements: 'Bank, building society or Post Office card account statements',
-            drivingLicence: 'UK photo driving licence',
-            wageSlip: 'Recent wage slip',
-            none: 'None of these options',
+            addressLine1: 'Address line 1',
```

Comment:
Could we just remove the answers here, and create the field labels in the page class / template? It feels less confusing to me than treating 'Address line 1' as a predefined answer in the same way as the others in this file.

Reply:
ah yeah I see what you mean, `answers` here are predefined answers only, will move them 👍